web/app.js

```diff
@@ -2764,6 +2771,20 @@ function onClick(evt) {
   }
 }
 
+function closeEditorUndoBar(evt) {
+  if (!this.editorUndoBar?.isOpen) {
+    return;
+  }
+  if (this.appConfig.secondaryToolbar?.toolbar.contains(evt.target)) {
+    this.editorUndoBar?.hide();
```

Comment:
Given the early return above, you can't reach this line unless the undo-bar is available.
```suggestion
    this.editorUndoBar.hide();
```